Objects/frameobject.c

```diff
@@ -649,6 +649,9 @@ frame_dealloc(PyFrameObject *f)
 static inline Py_ssize_t
 frame_nslots(PyFrameObject *frame)
 {
+    assert(frame->f_valuestack - frame->f_localsptr ==
+           // fastlocals + builtins + globals + locals
+           frame->f_code->co_nfastlocals + 3);
```

Comment:
OK, I see it is just misnamed, and the comment is misleading. Cell and free variables are also included.